code/modules/antagonists/protagonists/protagonist.dm

```diff
@@ -0,0 +1,39 @@
+/datum/antagonist/protagonist
+	name = "Protagonist"
+	roundend_category = "protagonists"
+	antagpanel_category = "Protagonists"
+	job_rank = ROLE_PROTAGONIST
+	show_to_ghosts = TRUE
+	///Some protagonists have min ages
+	var/min_age = 16
```

Comment:
our minimum age is 17, so prepare for intense heckling if you keep this

Reply:
it shouldn't matter then! :) you can't go lower this is just a minimum specific to the role, but if you can't even get 16 in prefs then you can't get 16 here. Its more for protags that might need to be above 30 for example.